### PR TITLE
Python 2 stub for csv

### PR DESCRIPTION
**Pull request by zhangwen0411**

### CODE REVIEW

File: stubs/2.7/csv.pyi [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    quotechar = ...  # type: str
+    escapechar = ...  # type: str
+    doublequote = ...  # type: bool
+    skipinitialspace = ...  # type: bool
+    lineterminator = ...  # type: str
+    quoting = ...  # type: int
+    def __init__(self) -> None: ...
+
+class excel(Dialect):
+    delimiter = ...  # type: str
```
Comment (JukkaL) [inline stubs/2.7/csv.pyi]: Many of the attributes are inherited; you can just leave them out from base classes so simplify the stub a bit. This also applies to some of the other `Dialect` derived classes.

File: stubs/2.7/csv.pyi [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+# Public interface of _csv.writer
+class Writer(metaclass=ABCMeta):
+    dialect = ...  # type: Dialect
+
+    @abstractmethod
+    def writerow(self, row: _Row) -> None: ...
+
+    @abstractmethod
+    def writerows(self, rows: List[_Row]) -> None: ...
```
Comment (JukkaL) [inline stubs/2.7/csv.pyi]: What does the documentation say about the argument, is a list or could be potentially use a sequence or an iterable?

**Comment by JukkaL** [comment]
Thanks, looks good! Just a few small things (see comments).

File: stubs/2.7/csv.pyi (hunk 2 above)
Comment (zhangwen0411) [inline stubs/2.7/csv.pyi]: The documentation says "a list of row objects as described above" so I took it to mean Python lists.

Comment (zhangwen0411) [inline stubs/2.7/csv.pyi]: Actually, the documentation also says `writer.writerows(someiterable)` so I'll just use iterable.

**Pull request merged**